.github/workflows/test_iree.yml

```diff
@@ -63,6 +65,8 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           python3 -m pip install -r iree_tests/requirements.txt
+          python3 -m pip install -r iree_special_models/requirements.txt
```

Comment:
Are these extra requirements needed in this job? This is just running onnx tests here

Reply:
Not needed, removed